### PR TITLE
(maint) Update changelog for 2.0.0 release

### PR DESCRIPTION
**Pull request by sarameisburger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.9.8 - 2019-03-12
```
Comment (mwaggett) [inline CHANGELOG.md]: This should be a 2.0.0 release, since removing those files is backwards-incompatible, according to @underscorgan.

Comment (sarameisburger) [reply CHANGELOG.md]: ok

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## 2.0.0 - 2019-03-12
+This is a major feature release.
```
Comment (sarameisburger) [inline CHANGELOG.md]: @mwaggett updated. Wasn't sure if this is what we want to call it tho

Comment (mwaggett) [reply CHANGELOG.md]: hmm, yeah. maybe `backwards-incompatible release`? since it's not really a new feature.. ¯\_(ツ)_/¯

**Comment by underscorgan** [comment]
hmm yeah before we do the 2.0 release we should at least remove the task that is definitely not still functional since we removed those packaging artifacts. We might want to see if there's anything else obvious that needs to happen.

**Comment by underscorgan** [comment]
https://github.com/puppetlabs/ezbake/pull/532 at least removes legacy-build

File: CHANGELOG.md (hunk 2 above)
Comment (sarameisburger) [reply CHANGELOG.md]: @underscorgan is there a specific wording i should be using? I looked at https://semver.org/ but wasn't sure

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+
+Feature:
+  * (RE-12093) Remove old rpm and deb artifacts previously used to build packages.
+  * Remove legacy build action that used the removed artifacts. Packages can only
+  be built using FPM.
+
```
Comment (sarameisburger) [inline CHANGELOG.md]: @underscorgan updated

File: CHANGELOG.md (hunk 2 above)
Comment (underscorgan) [reply CHANGELOG.md]: mm, yeah, probably backwards-incompatible release.

File: CHANGELOG.md (hunk 3 above)
Comment (underscorgan) [reply CHANGELOG.md]: can you switch legacy build to `legacy-build` for extra clarity?

**Comment by sarameisburger** [comment]
@underscorgan @mwaggett updated!

**Pull request merged**